### ISSUE TITLE
Doesn't Work with Enhanced Terrain Layer Version 1.0.39

### ISSUE THREAD

**Issue by SyraLessThanThree** [caewok/fvtt-elevation-ruler#8]
OS: Windows 10
Foundry Version: V9 - Build 269
Elevation Ruler: Version 0.4.0
Enhanced Terrain Layer: Version 1.0.39
System DnD5e
![2022-04-26 22-05-57_new](https://user-images.githubusercontent.com/25936043/165383684-77363206-ba9a-4881-8b35-ca7a4e2653f3.gif)
got this exception when Pressing F12
![image](https://user-images.githubusercontent.com/25936043/165384066-98eda0c6-8106-4858-a24e-d007616a401d.png)

**Comment by SyraLessThanThree** [comment]
![image](https://user-images.githubusercontent.com/25936043/165385546-4f038a5c-e3a4-4308-9f0d-51344edc19b0.png)

**Comment by Zamrod** [comment]
This also does not work in v10 with Enhanced Terrain Ruler 10.1. Any attempt to hold in control and then drag a ruler on the canvas gives the following error:

```
Uncaught TypeError: t.shape.contains is not a function
[Detected 3 packages: enhanced-terrain-layer, elevationruler, lib-wrapper]
    at terrainlayer.js:320:28
    at Array.filter (<anonymous>)
    at TerrainLayer.terrainFromPixels (terrainlayer.js:317:40)
    at TerrainLayerElevationAtPoint (terrain_elevation.js:178:35)
    at Ruler.terrainElevationAtPoint (terrain_elevation.js:118:29)
    at Ruler.elevationAtOrigin (terrain_elevation.js:38:21)
    at Ruler.terrainElevationAtPoint (terrain_elevation.js:89:94)
    at Ruler.terrainElevationAtDestination (terrain_elevation.js:73:15)
    at Ruler._getMeasurementSegmentsRuler (segments.js:22:45)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:620:16)
    at 🎁Ruler.prototype._getMeasurementSegments#0 (libWrapper-wrapper.js:189:20)
    at Ruler.measure (foundry.js:32124:26)
    at Ruler._onMouseMove (foundry.js:32464:12)
    at Canvas._onDragLeftMove (foundry.js:24285:42)
    at MouseInteractionManager.callback (foundry.js:27633:45)
    at MouseInteractionManager._handleDragMove (foundry.js:27990:17)
    at MouseInteractionManager._handleMouseMove (foundry.js:27962:22)
    at a.emit (index.js:202:33)
    at r.dispatchEvent (interaction.min.mjs:8:12235)
    at r.processPointerMove (interaction.min.mjs:8:17255)
    at t.recursiveFindHit (interaction.min.mjs:8:4098)
    at t.findHit (interaction.min.mjs:8:4157)
    at r.processInteractive (interaction.min.mjs:8:12841)
    at r.onPointerMove (interaction.min.mjs:8:16687)
```

**Comment by caewok** [comment]
Fixed in v0.6.6. Requires v10 of Enhanced Terrain Layer
